packages/editor/src/bindings/post-meta.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../store';
+
+const { getCurrentPostType } = select( editorStore );
+
+// Prettify the name until the label is available in the REST API endpoint.
+const keyToLabel = ( key ) => {
+	return key
+		.split( '_' )
+		.map( ( word ) => word.charAt( 0 ).toUpperCase() + word.slice( 1 ) )
+		.join( ' ' );
+};
+
+export default {
+	name: 'post_meta',
+	label: 'Post Meta',
```

Comment:
It needs to be translatable.

Reply:
Is using `__( 'Post Meta' )` enough?

Reply:
Yes.

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/58085/commits/96449467ec05e18806d6bf4dadb92a16feba0ec5).